### PR TITLE
Feature/fix isseues 6 7 8

### PR DESCRIPTION
**Pull request by skorodenko**

### CODE REVIEW

File: hangman.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class BaseVariables():
     """
     vowels = VOWELS_LIST
     hint = HINT_SYMBOL
```
Comment (Hozhyi) [inline hangman.py:26]: Duplication of constants

File: hangman.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ class BaseVariables():
...
-    gap = GAP_SYMBOL+" "
+    gap = GAP_SYMBOL + " "
```
Comment (Hozhyi) [inline hangman.py:27]: If `gap` is also a constant - name it accordingly. And could you find a better name for it? Especially when you have constant `GAP_SYMBOL` - it's not obvious the meaning of `GAP`

**Review by Hozhyi — CHANGES_REQUESTED** [review]

File: hangman.py (hunk 1 above)
Comment (skorodenko) [reply hangman.py:26]: Constant declaration moved into BaseVariables class.

File: hangman.py (hunk 2 above)
Comment (skorodenko) [reply hangman.py:27]: Resolved.

File: hangman.py [hunk 3]
```diff
@@ -192,7 +189,7 @@ def match_with_gaps(my_word, other_word, letters_guessed):
     """
     if len(my_word) == len(other_word):
         for i in range(len(my_word)):
-            if my_word[i] == BaseVariables.gap.strip():
+            if my_word[i] == BaseVariables.GAP_SYMBOL.strip():
```
Comment (Hozhyi) [inline hangman.py:192]: UNREVEALED_LETTER ?

**Review by Hozhyi — APPROVED** [review]

**Pull request merged**